src/ils_vns.jl

```diff
@@ -0,0 +1,251 @@
+#=
+# VNS for the RENAULT Roadef 2005 challenge
+# inspired by work of
+# Celso C. Ribeiro, Daniel Aloise, Thiago F. Noronha, Caroline Rocha, Sebastián Urrutia
+#
+# @Author Boualem Lamraoui, Benoît Le Badezet, Benoit Loger, Jonathan Fontaine, Killian Fretaud, Rémi Garcia
+# =#
+
+
+
+# Complete strategy can be summerize as follow :
+#
+#   (1) Use a constructive heuristic to find an initial solution.
+#   (2) Use an improvement heuristic to optimize the first objective.
+#   (3) Use an improvement heuristic to optimize the second objective.
+#   (4) Use a repair heuristic to make the current solution feasible.
+#   (5) Use an improvement heuristic to optimize the third objective.
+#
+
+
+
+##===================================================##
+##                 Data stuctures                    ##
+##===================================================##
+
```

Comment:
I suggest to create a solution structure instead of multiple global variable.
For example :
```julia
mutable struct Solution
sequence::Array{Int, 1}
M1::Array{Int, 2}
M2::Array{Int, 2}
M3::Array{Int, 2}
end
```

Reply:
```Julia
Solution(nCars, options) = (
sequence = zeros(Int, nCars),
#etc
)
```